xapi_bridge/statements/problem.py

```diff
@@ -52,7 +51,7 @@ def get_result(self, event):
         event_data = self.get_event_data(event)
         # for now we are going to assume one submission
         # TODO: when is that not true? is it ever not true? What problem types?
-        submission = event_data['submission'][event_data['submission'].keys()[0]]
+        submission = event_data['submission'][list(event_data['submission'].keys())[0]]
```

Comment:
Perhaps raise an error or at least log if there is more than one key in `event_data['sugmission']`?
I'm not familiar at all with the `xapi_bridge` package, but in general it seems if there do happen to be multiple keys and the wrong key is used, what's the downside? Silent bad data?